lib/stringify.js

```diff
@@ -19,6 +19,7 @@ var toISO = Date.prototype.toISOString;
 
 var defaults = {
     delimiter: '&',
+    doNotEncodeKeys: false,
```

Comment:
I agree, encodeValuesOnly is better. I don't like negative option names. I'll update it to that